core/spec/models/spree/order_spec.rb

```diff
@@ -594,6 +594,14 @@ def merge!(other_order, user = nil)
         order = create(:order, state: "delivery")
         expect{ order.restart_checkout_flow }.to change{ order.state }.from("delivery").to("cart")
       end
+
+      it "doesn't add errors to the order" do
+        order = create(:order, state: "delivery", line_items: [])
```

Comment:
What do you think about moving this line in a `let` so that we can share the same `order` definition with both specs inside this `context`? This would also remove the confusion about the different approaches for creating the empty order in delivery state (this line is explicit about empty line items, while in the previous spec, this remains implicit).

Reply:
Changed!